app/services/csv/sales_log_csv_service.rb

```diff
@@ -129,31 +131,38 @@ def label_if_boolean_value(value)
       return "No" if value == false
     end
 
-    ATTRIBUTE_MAPPINGS = {
-      "saledate" => %w[day month year],
-      "exdate" => %w[exday exmonth exyear],
-      "hodate" => %w[hoday homonth hoyear],
-      "postcode_full" => %w[pcode1 pcode2],
-      "ppostcode_full" => %w[ppostc1 ppostc2],
-      "la" => %w[la la_label],
-      "prevloc" => %w[prevloc prevloc_label],
-      "assigned_to_id" => %w[assigned_to],
-      "owning_organisation_id" => %w[owning_organisation_name],
-      "managing_organisation_id" => %w[managing_organisation_name],
-      "value" => %w[value value_value_check],
-      "mscharge" => %w[mscharge mscharge_value_check],
-    }.freeze
+    def attribute_mappings
+      mappings = {
+        "saledate" => %w[day month year],
+        "exdate" => %w[exday exmonth exyear],
+        "hodate" => %w[hoday homonth hoyear],
+        "postcode_full" => %w[pcode1 pcode2],
+        "ppostcode_full" => %w[ppostc1 ppostc2],
+        "la" => %w[la la_label],
+        "prevloc" => %w[prevloc prevloc_label],
+        "assigned_to_id" => %w[assigned_to],
+        "owning_organisation_id" => %w[owning_organisation_name],
+        "managing_organisation_id" => %w[managing_organisation_name],
+        "value" => %w[value value_value_check],
+        "mscharge" => %w[mscharge mscharge_value_check],
+      }
+      if @user.support? && @year >= 2024
+        mappings["beds"] = ORDERED_ADDRESS_FIELDS + %w[beds]
```

Comment:
I agree that I am surprised by this.
The ticket specifies that they should be placed after the existing address fields, but then it says put it directly after/before {field that doesn't sound addressy to me}
I think I'm missing some context here as to changes since I left the project of what info we take from the user, what we get back from the address API and what we display in the csv export.